apps/src/templates/manageStudents/ConfirmRemoveStudentDialog.jsx

```diff
@@ -54,18 +55,15 @@ export default class ConfirmRemoveStudentDialog extends React.Component {
           {hasEverSignedIn &&
             <div>
               <p>
-                <strong>{i18n.removeStudentBody1()}</strong>
-                {' '}
-                {i18n.removeStudentBody2()}
-                {' '}
-                <a href={RELEASE_OR_DELETE_RECORDS_EXPLANATION} target="_blank">
```

Comment:
Note that with this change, we're losing the `target="_blank"` functionality of the learn more link. Should probably confirm whether or not that's okay